tools/dockerfile/grpc_clang_format/clang_format_all_the_things.sh

```diff
@@ -29,7 +29,7 @@ for dir in $DIRS
 do
   for glob in $GLOB
   do
-    files="$files `find ${CLANG_FORMAT_ROOT}/$dir -name $glob -and -not -name '*.generated.*' -and -not -name '*.pb.h' -and -not -name '*.pb.c' -and -not -name '*.pb.cc' -and -not -name '*.pbobjc.h' -and -not -name '*.pbobjc.m' -and -not -name '*.pbrpc.h' -and -not -name '*.pbrpc.m' -and -not -name end2end_tests.cc -and -not -name end2end_nosec_tests.cc -and -not -name public_headers_must_be_c89.c`"
+    files="$files `find ${CLANG_FORMAT_ROOT}/$dir -name $glob -and -not -name '*.generated.*' -and -not -name '*.pb.h' -and -not -name '*.pb.c' -and -not -name '*.pb.cc' -and -not -name '*.pbobjc.h' -and -not -name '*.pbobjc.m' -and -not -name '*.pbrpc.h' -and -not -name '*.pbrpc.m' -and -not -name end2end_tests.cc -and -not -name end2end_nosec_tests.cc -and -not -name public_headers_must_be_c89.c -and -not -name stats_data.h -and -not -name stats_data.cc`"
```

Comment:
Why lose the clang fmt on the generated code? It makes changes easier to see if someone is not familiar with how changes from the yaml file are translated to changes in the cc code.

The other exclusions make sense here, since they are generated, but rather readable. Unless this affects binary size... that could convince me

Reply:
Done! 
(I seemed to recall a reason which required generated files to be added here, but it looks like did not face the issue here.)